plugins/Deflect/DeflectParameters.cpp

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2015-2018, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "DeflectParameters.h"
+#include "utils.h"
+
+#include <deflect/Stream.h>
+
+namespace brayns
+{
+PropertyMap DeflectParameters::createPropertyMap()
+{
+    PropertyMap properties("Deflect plugin parameters");
+    properties.setProperty(
+        {PARAM_ENABLED, true,
+         Property::MetaData{"Enable streaming", "Enable/disable streaming"}});
+    properties.setProperty(
+        {PARAM_ID, std::string(),
+         Property::MetaData{
+             "Stream ID",
+             "The ID/name of the stream, equivalent to DEFLECT_ID"}});
+    properties.setProperty(
+        {PARAM_HOSTNAME, std::string(),
+         Property::MetaData{"Stream hostname", "Hostname of Deflect server"}});
+    properties.setProperty(
+        {PARAM_PORT,
+         (int32_t)deflect::Stream::defaultPortNumber,
+         {1023, 65535},
```

Comment:
The range should start at 1, I don't see any reason to prevent running Deflect server on a privileged port (which doesn't necessarily imply it runs as a privileged process)

Reply:
fair enough